src/main/scala/com/softwaremill/realworld/articles/Article.scala

```diff
@@ -0,0 +1,30 @@
+package com.softwaremill.realworld.articles
+
+import java.time.ZonedDateTime
+
+case class Article(
+    slug: String,
+    title: String,
+    description: String,
+    body: String,
+    tagList: List[String],
+    createdAt: ZonedDateTime,
+    updatedAt: ZonedDateTime,
+    favorited: Boolean,
+    favoritesCount: Int,
+    author: ArticleAuthor
+)
+
+case class StoredArticle(
+    id: Int,
```

Comment:
Sth for the future: introduce an opaque type for ids

Reply:
I will add it in the next PR if it will be still needed.